src/layout/flex-grid/src/index.tsx

```diff
@@ -0,0 +1,123 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx } from 'theme-ui'
+
+const DEFAULT_GUTTER_SIZE = 'xs'
+
+const getSpaceValue = (key: string) => (theme: any = {}) =>  theme.space[key]
+
+const getNegativeSpaceValue = (key: string) => (theme: any = {}) => `-${getSpaceValue(key)(theme)}px`
+
+const getGutterSize = (theme: any) => theme?.grid?.sizes?.gutter || DEFAULT_GUTTER_SIZE
```

Comment:
do we not want to be able to specify different gutter on different devices?

Reply:
this allows you to do that (use an array)